pom.xml

```diff
@@ -53,6 +54,12 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-XX:MaxPermSize=${permgen.size}</argLine>
```

Comment:
This drops the `-Djava.awt.headless=true` that is inherited from the parent pom.  Is that intentional?